load_tests/load_test.py

```diff
@@ -135,18 +135,26 @@ def generate_task_definition(throughput, input_logger, s3_fluent_config_arn):
         elif(key == OUTPUT_PLUGIN):
             for sub_key in task_definition_dict[key]:
                 data = data.replace(sub_key, task_definition_dict[key][sub_key])
-    fout = open(f'./load_tests/task_definitions/{OUTPUT_PLUGIN}_{throughput}.json', 'w')
-    fout.write(data)
-    fout.close()
-    fin.close()
 
-    os.system(f'aws ecs register-task-definition --cli-input-json file://load_tests/task_definitions/{OUTPUT_PLUGIN}_{throughput}.json {(">/dev/null", "")[IS_TASK_DEFINITION_PRINTED]}')
+    # Register task definition
+    task_def = json.loads(data)
+    
+    if IS_TASK_DEFINITION_PRINTED:
+        print("Registering task definition:")
+        print(json.dumps(task_def, indent=4))
+        session.client('ecs').register_task_definition(
+            **task_def
+        )
+    else:
+        print("Registering task definition")
```

Comment:
does it mean we don't register the task definition in this case?

Reply:
Great catch! Thank you! Changed.